src/Sylius/Component/Payment/Model/PaymentMethod.php

```diff
@@ -144,8 +142,6 @@ public function getName()
     public function setName($name)
     {
         $this->name = $name;
-
-        return $this;
```

Comment:
As @michalmarcinkowski wrote in my previous PR, we no longer do fluent interfaces in Sylius.